pipestat/parsed_schema.py

```diff
@@ -50,13 +50,27 @@ class Config:
     return BaseModel
 
 
-def _safe_pop_one_mapping(key: str, data: Dict[str, Any], info_name: str) -> Any:
-    value = data.pop(key, NULL_MAPPING_VALUE)
-    if isinstance(value, Mapping):
-        return value
-    raise SchemaError(
-        f"{info_name} info in schema definition has invalid type: {type(value).__name__}"
-    )
+def _safe_pop_one_mapping(
+    mappingkey: str, data: Dict[str, Any], info_name: str, subkeys: Optional[List[str]] = None
+) -> Any:
+    """
+    subkeys exist if the output schema is in a JSON schema format
```

Comment:
This docstring doesn't make sense to me.

Reply:
Updated to be more explicit.